lib/plugins/aws/package/compile/events/alb/lib/listenerRules.js

```diff
@@ -13,6 +13,82 @@ module.exports = {
         event.multiValueHeaders
       );
 
+      let Order = 0;
+      const Actions = [];
```

Comment:
Let's not capitalize this var names. I know it's put uppercase to match names in CF config, still in terms of JS variables, capitalized var names are used for function constructors, so using it for primitives or arrays raises confusion

Reply:
The same thing is done with Condition so thought to follow suite...

Reply:
Yes, sorry about that. I know the conventions do not always align across the project.

It's time-taking task to refactor everything to be in line. We try our best to ensure all newly introduced code matches intended convention.